packages/truffle-db/src/loaders/artifacts/test/index.ts

```diff
@@ -39,14 +55,18 @@ const config = {
 };
 
 const compilationConfig =  {
-    contracts_directory: path.join(__dirname, "compilationSources"),
-    contracts_build_directory: path.join(__dirname, "sources"),
-    all: true
+  contracts_directory: path.join(__dirname, "compilationSources"),
+  contracts_build_directory: path.join(__dirname, "sources"),
+  all: true
 }
 
 const db = new TruffleDB(config);
 const Migrations = require(path.join(fixturesDirectory, "Migrations.json"));
-const Build = [require(path.join(__dirname, "sources", "MagicSquare.json")), require(path.join(__dirname, "sources", "Migrations.json")), require(path.join(__dirname, "sources", "SquareLib.json")) ];
+const Build = [
```

Comment:
Naming seems a bit off here: for one, it should probably be lowercase because it's a specific thing, not a type of thing or module of some kind.

Can we just call it `artifacts`? I don't think this file uses that name for something else, and that's what this list is.

Reply:
Agree! Changing.